Application/Dtos/Blog/PostDetailsDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Application.Dtos.Blog
+{
+    public class PostDetailsDto
+    {
+        public Guid Id { get; set; }
+        public string Message { get; set; }
+        public string Title { get; set; }
+        public List<CommentVm> Comments { get; set; }
```

Comment:
Comment should be its separate entity and you should have a list of commentIds

Reply:
why?
this will complicate the displaying logic at the front in my case.

Reply:
DTOs are for data transfer as you said. How does data transfer complicate displaying logic? That means you have unnecessary cohesion in your logic.

Reply:
I get this dto from the database, then I perform some actions and create a view model, then send it to the front.